views/gem.erb

```diff
@@ -9,8 +9,8 @@
         </h2>
         <ul class="pulls">
           <% gem.fetch(:pull_requests).each do |pull_request| %>
-            <li class="pull">
-              <a href="<%= pull_request.fetch(:url) %>">
+            <li class="pull <%= old_pull_request?(pull_request.fetch(:opened_at))? 'warning' : '' %>">
```

Comment:
Could we get away with adding the class to just one element?

Reply:
the css for `a` being set to white overwrote the warning class, and it felt off to have the time-elapsed being white which is what happens if I set it to just the `li` - hence it being in both places